liquibase-core/src/main/java/liquibase/diff/output/changelog/core/MissingTableChangeGenerator.java

```diff
@@ -36,7 +36,7 @@ public static void setDefaultValue(ColumnConfig columnConfig, Column column, Dat
         Object defaultValue = column.getDefaultValue();
         if (defaultValue == null) {
             // do nothing
-        } else if (column.isAutoIncrement()) {
+        } else if (column.isAutoIncrement() && defaultValue == null) {
```

Comment:
@nvoxland This feels like it might reach too far, do you think I should constrain this change to only apply to Postgres databases?